05.User-resource.md

```diff
@@ -114,6 +114,36 @@ Try it from the CLI:
 
 ### Unread Items
 
+You can retrieve unread items and mentions using the following endpoint.
+
+```
+GET /v1/user/:userId/rooms/:roomId/unreadItems
+```
+
+```
+{
+	"chat": [
+		"560ae5f495756f1402bc841d", 
+		"560aeef6f4b61c106fb2a9f3",
+		"560aef2b552ed7913279df52"
+	],
+	"mention": [
+		"560b167a967c1bad7852bc57"
+	]
+}
+```
+
+Try it from the CLI:
+```
+'demo api-access';
+{
+    "verb": "get",
+    "resource": "{{api_url}}/v1/user/:userId/rooms/:roomId/unreadItems"
+}
+```
+
+### Mark Unread Items
+
 Is worth mentioning there's an additional endpoint nested under rooms that you can use to mark chat messages as read.
```

Comment:
Not specific to your contribution, but if you could fix this error and update to: "It is worth mentioning that there is..."